qa/rbd/conf/mon.yaml

```diff
@@ -0,0 +1,6 @@
+overrides:
+  ceph:
+    conf:
+      mon:
+        # cephadm can take up to 5 minutes to bring up remaining mons
+        mon down mkfs grace: 300
```

Comment:
This seems like it would affect not just NVMeOF but all of RBD tests.  In `qa/tasks/ceph.conf.template`, it's set to two minutes (bumped from the default of one minute) and I don't see it being overridden in any other suite:
- if it's just NVMeOF tests doing something that necessitate this, make it specific to NVMeoF by moving under `qa/suites/rbd/nvmeof`
- if the premise of the change is that all tests would benefit from this, provide a more detailed explanation and submit a separate PR changing `qa/tasks/ceph.conf.template` which should be picked up by all suites

@VallariAg ^^

Reply:
Understood, I'll look into this.